include/deal.II/optimization/line_minimization.h

```diff
@@ -179,9 +179,9 @@ namespace LineMinimization
    * @code
    *   auto func = [](const double x)
    *   {
-   *     const double f = 100. * std::pow(x, 4) + std::pow(1. - x, 2); // Value
-   *     const double g = 400. * std::pow(x, 3) - 2. * (1. - x); // Gradient
-   *     return std::make_pair(f, g);
+   *     const double f = 100. * Utilities::fixed_power<2>(x, 4) + std::pow(1. -
+   * x); // Value const double g = 400. * Utilities::fixed_power<3>(x) - 2. *
+   * (1. - x); // Gradient return std::make_pair(f, g);
```

Comment:
I think this got jumbled up.

Reply:
Ah, bummer, I had missed that. Fixed now.